src/Symfony/Component/VarExporter/Internal/LazyObjectRegistry.php

```diff
@@ -76,14 +76,6 @@ public static function getClassResetters($class)
             }, null, $scope);
         }
 
-        $resetters[] = static function ($instance, $skippedProperties) {
-            foreach ((array) $instance as $name => $value) {
-                if ("\0" !== ($name[0] ?? '') && !\array_key_exists($name, $skippedProperties)) {
-                    unset($instance->$name);
```

Comment:
This unset is meant for dynamic properties, but we need that part only when resetting objects, not when creating them. Code moved to the reset-only code path.